modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1257,6 +1259,14 @@ Where `<expr>` is any Cypher expression and `<TYPE>` is one of the following typ
 * `ZONED DATETIME`
 * `DURATION`
 * `POINT`
+* `NODE`
+* `RELATIONSHIP`
+* `MAP`
+* `LIST<INNER_TYPE>`
+* `PATH`
+* `PROPERTY VALUE`
+* `ANY`
+* `<TYPE> NOT NULL`
 
 For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```

Comment:
We definitely need to update `values-and-types/property-structural-constructed.adoc` to say things about `NOT NULL` as it is currently missing but should be covered as part of the type system.

There is also a sentence in there with a TODO that can be acted on now that the expressions allow all types listed there.

Reply:
I can definitely join you on this part if you want

Reply:
speaking of this file, we need to update that list of types and synonyms with the newly introduced types as well (it has some of them such as map and node but not all, for example missing nothing and any. it's just covering the types already mentioned in the types chapter and not the things that are now official types that didn't exist previously.

Reply:
I updated it a little, but we can go over it together tomorrow if you are free :)